lib/rubocop/cop/style/ascii_comments.rb

```diff
@@ -5,6 +5,13 @@ module Cop
     module Style
       # This cop checks for non-ascii (non-English) characters
       # in comments.
+      #
+      # @example
+      #   # bad
+      #   # Translates from English to 日本語。# rubocop:disable Style/AsciiComments
```

Comment:
I guess you can disable this cop for the entire file, so this wouldn't appear in the documentation.